test/test_methods_test.rb

```diff
@@ -11,6 +11,18 @@ def test_asserts_controller_submits_correct_form
     end
   end
 
+  def test_asserts_controller_submits_at_all
+    runner = FooControllerTest.new :foo
+
+    error = assert_raises MiniTest::Assertion do
+      runner.assert_submits FooForm do
+      end
+    end
+
+    assert_equal "Expected TestMethodsTest::FooController to submit a TestMethodsTest::FooForm",
+                 error.message
```

Comment:
Good question… I'm a little hesitant to say yes, but having a large chunk of text as a hard-coded argument is always weird. The style-guide could be "anytime there is a hard-coded string that is longer than 80 characters or spans multiple lines, use a heredoc"—maybe there should be more qualifiers than that?